test/e2e/apps/disruption.go

```diff
@@ -54,20 +61,39 @@ var _ = SIGDescribe("DisruptionController", func() {
 		ns = f.Namespace.Name
 	})
 
+	ginkgo.Context("Listing PodDisruptionBudgets for all namespaces", func() {
+		anotherFramework := framework.NewDefaultFramework("disruption-2")
+
+		ginkgo.It("should list and delete a collection of PodDisruptionBudgets", func() {
+			specialLabels := map[string]string{"foo_pdb": "bar_pdb"}
+			labelSelector := "foo_pdb=bar_pdb"
```

Comment:
You can use `labels.SelectorFromSet` here to turn the labels into a selector.

Reply:
Good to know. Done!